Schedule.API/Controllers/ExaminationController.cs

```diff
@@ -90,5 +96,19 @@ public IActionResult AdvancedSearch(ExaminationAdvancedFilterDto dto)
             dto.PatientId = Int32.Parse(HttpIdentityHandler.GetUserIdFromRequest(HttpContext.Request));
             return Ok(_examinationService.Search(dto));
         }
+
+        [HttpPost]
+        [Route("check-rooms")]
+        public IActionResult GetUnavailableRooms(EquipmentRelocationDto dto)
+        {
+            return Ok(_equipmentRelocationSchedulingService.GetUnavailableRooms(dto));
+        }
```

Comment:
The Route is not suitable for this method

Reply:
I renamed it to unavailable-rooms.